pkg/driver/node_test.go

```diff
@@ -303,6 +303,19 @@ func TestNodePublishVolume(t *testing.T) {
 			mountArgs:     []interface{}{volumeId + ":/", targetPath, "efs", []string{}},
 			mountSuccess:  true,
 		},
+		{
+			name: "success: normal with volume context populated from dynamic provisioning",
```

Comment:
I would like to see an e2e test for cross Account mount but it's going to be hard since we need 2 accounts and to store creds somewhere. Any ideas how to achieve this? we could store the creds in secrets manager in the existing CI account and construct the secret by retrieving fro msecrets manager as part of the test.

Reply:
Need to give this some thought. Apart from having to manage credentials for two accounts, Cross-account mount requires `vpc peering`. I'm not sure how to achieve this on the fly for the integ tests.